portfolio/src/main/webapp/script.js

```diff
@@ -14,7 +14,7 @@
 
 // Fetches data from servelet when the document is loaded
 window.onload = async () => {
-  const response = await fetch('/data');
+  const response = await fetch('/comment');
   const jsonArray = await response.json();
```

Comment:
Give this a more descriptive name like `commentsJson`

Reply:
Will fix this branch in js branch